src/Http/Headers/src/MediaTypeHeaderValue.cs

```diff
@@ -205,7 +206,7 @@ public IList<NameValueHeaderValue> Parameters
         /// </summary>
         public double? Quality
         {
-            get { return HeaderUtilities.GetQuality(_parameters); }
+            get { return HeaderUtilities.GetQuality(Parameters); }
```

Comment:
Wasn't this _parameters to avoid having Parameters allocate if _parameters was null, and GetQuality was null tolerant?

Reply:
Nice catch. I missed addressing this from the original PR: https://github.com/dotnet/aspnetcore/pull/18084/files#r362617412